static/inline/js/index.js

```diff
@@ -0,0 +1,36 @@
+;(function(window,document) {
+    var render = function(target,data){
+        if (!target) return;
+        var frag = document.createDocumentFragment();
+        var main = document.createElement('div');
+        var d = data || {};
+
+        main.id = d.id || '';
+        main.classList.add('iframe-content');
+        main.innerHTML = data.inlineTemplate;
+
+        frag.appendChild(main);
+        target.appendChild(frag);
+    };
+
+    var body = document.body;
+
+    var initialize = function(){        
+        var settings = {};
+
+        if (Utils.isset(parent) && Utils.isset(parent,'__TVPage__') && Utils.isset(parent.__TVPage__, 'config')) {
+            settings = parent.__TVPage__.config[body.getAttribute('data-id')];
+        }
+        var inlineSettings = JSON.parse(JSON.stringify(settings));
+
+        render(body,{
+            id: settings.name,
+            title: settings.title || 'Recommended Videos',
+            inlineTemplate: settings.templates.inline
+        });
+
+        Inline(settings.name, inlineSettings);
+    };
+    
+    initialize();
```

Comment:
I can see that you removed the polling/check for globals, like we do in other templates
You will need this, I have seen the widget failing some other time when I paste the url and I do not have the devTools/console open.

Failures are TVPage or _tvpa globals undefined..

This is ocassionated by the approach we use to create the iframes